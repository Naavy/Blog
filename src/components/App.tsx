import React from 'react';
import Users from './users';
import Posts from './posts';
import ChangeUserName from './change-username';
import logo from '../static/images/logo.png';
import styled from 'styled-components';
import { Grid } from '@material-ui/core';

const Logo = styled.div`
  display: flex;
  justify-content: center;
`;

const Container = styled.div`
  margin: 40px;
  display: flex;
`;

export class App extends React.Component<{}, {}> {
  render () {
    return (
      <React.Fragment>
        <Logo>
          <img src={logo} alt='logo' style={{ width: '200px' }} />
        </Logo>
        <Container>
          <Grid item sm={6}>
            <Users />
            <ChangeUserName />
          </Grid>
          <Grid item sm={4} >
            <Posts />
          </Grid>
        </Container>
      </React.Fragment>
    )
  };
};

export default App;