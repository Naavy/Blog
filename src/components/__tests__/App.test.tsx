import React from 'react';
import { shallow } from 'enzyme';
import { App } from '../App';

describe('App', () => {
  it('should render', () => {
    // when
    const wrapper = shallow(<App />);

    // then
    expect(wrapper).toMatchSnapshot();
  });

});