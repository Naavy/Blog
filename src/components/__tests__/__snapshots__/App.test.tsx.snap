// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search-User should render 1`] = `ShallowWrapper {}`;
